src/schema/rules/sidecars/eyetrack.yaml

```diff
@@ -0,0 +1,64 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+---
+EyeTrackingRecommended:
+  selectors:
+    - modality == "eyetrack"
+    - datatype == "eyetrack"
+    - suffix == "eyetrack"
+  fields:
+    Manufacturer:
+      level: recommended
+      description: |
+        The sampling frequency of data channels that deviate from the main
+         sampling frequency SHOULD be specified in the `channels.tsv` file.
+    ManufacturersModelName:
+      level: recommended
+      description: |
+        Manufacturer's designation of the eye-tracker model (for example"
+         `Eye-link 1000`).
+    SoftwareVersions: recommended
+    DeviceSerialNumber: recommended
+EyeTrackingRequired:
+  selectors:
+    - modality == "eyetrack"
+    - datatype == "eyetrack"
+    - suffix == "eyetrack"
+  fields:
+    SamplingFrequency: required
+    SampleCoordinateUnits: required
+    SampleCoordinateSystem: required
+    EnvironmentCoordinates: required
+    ScreenSize: required
+    ScreenResolution: required
+    ScreenDistance: required
+EyeTrackingMoreRecommended:
```

Comment:
```suggestion
    ScreenDistance: required

EyeTrackingMoreRecommended:
```

Reply:
I actually combined now `EyeTrackingMoreRecommended` with `EyeTrackingRecommended` but put the space with this part and `EyeTrackingRequire`.